uiowa/uiowa
# Add character limit to uiowa_statistic prefix/suffix fields

Resolves #2258, resolves #2240 

# Test

- Sync a site, `blt frontend`
- Edit an existing stat block and create new stat blocks.
- Test that you cannot enter a prefix and/or suffix greater than 3 characters.
- For existing stat blocks with greater than 3 characters in either/both fields, they remain unchanged, but upon opening cannot edit/save until the new limit is enforced.
- Check the rearrangement, styling of the prefix/title/suffix field combo, and whether there are any hiccups or concerns
